### PR TITLE
Add no timeouts option

### LINKED ISSUE #119: It does not accept --no-timeouts
It is not able to use no-timeouts parameter

### PR DESCRIPTION
**Pull request by furstenheim**
Fixes #119

### CODE REVIEW

File: bin-helper.js [hunk 1]
```diff
@@ -41,6 +41,11 @@ export default function binHelper(options) {
         extensions.push(ext);
     });
 
+    // --no-timeouts option
+    if (typeof options.timeouts === 'boolean') {
```
Comment (1999) [inline bin-helper.js]: That's probably the only thing which seems weird for me: is it mocha which translates --no-timeouts to "timeouts"?

Comment (furstenheim) [reply bin-helper.js]: I think it is done by yargs https://github.com/yargs/yargs/blob/master/docs/tricks.md#negate

Comment (1999) [reply bin-helper.js]: Gotcha

**Comment by 1999** [comment]
Really nice PR. Thank you!

**Comment by 1999** [comment]
What's your name btw? So I can add you to contributors list. @furstenheim

**Comment by furstenheim** [comment]
Thanks!! It is Gabriel Fürstenheim

**Pull request merged**